spec/features/movies/index_spec.rb

```diff
@@ -53,10 +53,9 @@
         fill_in :search, with: ''
 
         expect(current_path).to eq(movies_path)
-        expect(page).to have_content('40 Movies')
         expect(page).to have_css(".movie", count: 40)
       end
-# DO WE NEED THESE TESTS IF ALREADY TESTS IN FACADE SPEC?
```

Comment:
I believe it's still necessary as a feature test. The facade test doesn't confirm functionality on the view